lib/assert.js

```diff
@@ -122,6 +122,14 @@ function innerFail(obj) {
   throw new AssertionError(obj);
 }
 
+/**
+ * @param {any} actual
+ * @param {any} expected
+ * @param {string | Error} [message]
+ * @param {string} [operator]
+ * @param {Function} [stackStartFn]
+ * @returns {void}
```

Comment:
```suggestion
 * @returns {never}
```

This always throws so it never returns void even